analyzers/ruby/integration_test.go

```diff
@@ -96,6 +96,6 @@ var projects = []fixtures.Project{
 	fixtures.Project{
 		Name:   "vagrant",
 		URL:    "https://github.com/hashicorp/vagrant",
-		Commit: "c1b695c47119f4294d92152f0c8f8da27f657660",
+		Commit: "f36ece40e227539c0f07e52c56c617bae6586d6c",
```

Comment:
It's not relevant to the PR, but it's relevant to fixing the ruby acceptance tests so that I can merge it. I don't know the root cause, but for some reason cloning the old commit, hash started to only download directories and not the files needed to scan such as `Gemfile`. I don't know why this was happening in CI, but it was consistently failing.